migrations/20190919-initdb-banners.js

```diff
@@ -36,7 +36,8 @@ module.exports = {
                 { transaction }
             );
 
-            await queryInterface.addConstraint(table, ['message', 'createTime', 'type'], {
+            await queryInterface.addConstraint(table, {
+                fields: ['message', 'createTime', 'type'],
```

Comment:
[nits]
I feel that the field key should be placed under the name key because it fits with the other settings.

Reply:
fixed 🙆‍♂️